client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -204,24 +208,61 @@ class ProductForm extends Component {
 						label={ translate( 'Price' ) }
 						component={ renderPriceField }
 					/>
-					<ReduxFormFieldset name="multiple" type="checkbox" component={ CompactFormToggle }>
-						{ translate( 'Allow people to buy more than one item at a time.' ) }
-					</ReduxFormFieldset>
-					<ReduxFormFieldset
-						name="email"
-						label={ translate( 'Email' ) }
-						explanation={ translate(
-							'This is where PayPal will send your money.' +
-								" To claim a payment, you'll need a {{paypalLink}}PayPal account{{/paypalLink}}" +
-								' connected to a bank account.',
-							{
-								components: {
-									paypalLink: <ExternalLink href="https://paypal.com" target="_blank" />,
-								},
-							}
-						) }
-						component={ FormTextInput }
-					/>
+					{ config.isEnabled( 'memberships' ) && (
+						<ReduxFormFieldset name="recurring" type="checkbox" component={ CompactFormToggle }>
+							{ translate( 'Make this product a recurring subscription.' ) }
+						</ReduxFormFieldset>
+					) }
+					{ ! this.props.isRecurringSubscription && (
+						<div>
+							<ReduxFormFieldset name="multiple" type="checkbox" component={ CompactFormToggle }>
+								{ translate( 'Allow people to buy more than one item at a time.' ) }
+							</ReduxFormFieldset>
+							<ReduxFormFieldset
+								name="email"
+								label={ translate( 'Email' ) }
+								explanation={ translate(
+									'This is where PayPal will send your money.' +
+										" To claim a payment, you'll need a {{paypalLink}}PayPal account{{/paypalLink}}" +
+										' connected to a bank account.',
+									{
+										components: {
+											paypalLink: <ExternalLink href="https://paypal.com" target="_blank" />,
+										},
+									}
+								) }
+								component={ FormTextInput }
+							/>
+						</div>
+					) }
+					{ this.props.isRecurringSubscription && (
+						<div>
+							<QueryMembershipsConnectedAccounts />
+							<ReduxFormFieldset
+								name="stripe_account"
+								explanation={ translate(
+									'This is the Stripe Account where the funds will end up.'
+								) }
+								label={ translate( 'Stripe Account' ) }
+								component={ FormSelect }
+								children={ Object.values( this.props.membershipsConnectedAccounts ).map( acct => (
+									<option value={ acct.connected_destination_account_id }>
+										{ acct.payment_partner_account_id }
+									</option>
+								) ) }
+							/>
+							<ReduxFormFieldset
+								name="renewal_schedule"
+								explanation={ translate( 'After what time should the subscription renew?' ) }
+								label={ translate( 'Renewal Schedule' ) }
+								component={ FormSelect }
+							>
+								<option value="1 week">{ translate( '1 Week' ) }</option>
+								<option value="1 month">{ translate( '1 Month' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'month' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/new/memberships-v1)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).